pom.xml

```diff
@@ -369,7 +369,7 @@
                             <IMAGEMAGICK_EXE>/usr/bin/convert</IMAGEMAGICK_EXE>
                             <IMAGEMAGICK_DYN>/usr/lib64/ImageMagick-7.0.10/lib</IMAGEMAGICK_DYN>
                             <IMAGEMAGICK_ROOT>/usr/lib64/ImageMagick-7.0.10</IMAGEMAGICK_ROOT>
```

Comment:
Also this ?

Reply:
These are for imagemagik, not LO

Reply:
Ah, sorry - my mistake (I just saw "7.0"). Busy day 🤦